stack/dashboards/rpm/wazuh-dashboards.spec

```diff
@@ -0,0 +1,420 @@
+# Wazuh package SPEC
+# Copyright (C) 2015-2022, Wazuh Inc.
```

Comment:
Change all Copyright headers to set the year of the creation of such file. e.g. 2021